spec/helpers/users_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+# require 'rails_helper'
```

Comment:
This file is completely commented out, so shouldn't really be included in source control